docs/testing/debug-add-ins-using-f12-tools-ie.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Debug add-ins using developer tools for Internet Explorer
+description: 'Debug add-ins using the developer tools in Internet Explorer'
+ms.date: 10/20/2021
+ms.localizationpriority: medium
+---
+
+# Debug add-ins using developer tools in Internet Explorer
+
+This article shows how to debug the client-side code (JavaScript or TypeScript) of your add-in when the following conditions are met:
+
+- You cannot, or don't wish to, debug using tools built into your IDE; or you are encountering a problem that only occurs when the add-in is run outside the IDE.
+- Your computer is using a combination of Windows and Office versions that use the Internet Explorer webview control, Trident.
+
+To determine which browser is being used on your computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
+
+> [!TIP]
+> [!INCLUDE[Identify the webview through the add-in UI](../includes/identify-webview-in-ui.md)]
+
+> [!NOTE]
+> To install a version of Office that uses the Internet Explorer webview, see [Install a version of Office that uses Internet Explorer](#install-a-version-of-office-that-uses-internet-explorer).
+
+## Debug a task pane add-in using the F12 tools
+
+Windows 10 and 11 include a web development tool called "F12" because it was originally launched by pressing F12 in Internet Explorer. It is now an independent application that you can use to debug your add-in when it is running in the Internet Explorer webview control, Trident. The application is not available in earlier versions of Windows.
```

Comment:
```suggestion
Windows 10 and 11 include a web development tool called "F12" because it was originally launched by pressing F12 in Internet Explorer. It's now an independent application that you can use to debug your add-in when it's running in the Internet Explorer webview control, Trident. The application isn't available in earlier versions of Windows.
```